audit/projects/k8s-staging-addon-manager/buckets/artifacts.k8s-staging-addon-manager.appspot.com/metadata.txt

```diff
@@ -11,8 +11,8 @@ gs://artifacts.k8s-staging-addon-manager.appspot.com/ :
 	Labels:				None
 	Default KMS key:		None
 	Time created:			Wed, 30 Sep 2020 15:49:57 GMT
-	Time updated:			Thu, 09 Sep 2021 15:22:40 GMT
-	Metageneration:			11
+	Time updated:			Fri, 24 Sep 2021 20:11:44 GMT
+	Metageneration:			12
```

Comment:
`ensure_gcs_bucket_logging` still doesn't seem to be fully idempotent... this is from a run of `ensure-staging-storage.sh` I did across all staging projects to ensure everything was up to date